Validation/Geometry/test/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.root
```

Comment:
why the one in SimTracker/TrackerMaterialAnalysis/test and this no remains a mystery to me.
Since I have no power to un-hold, I have to remove it.